### ISSUE TITLE
Dashboard: Focus does not go to the correct location after clicking Submit and Apply

### ISSUE THREAD

**Issue by qanitah** [mmiron-HC/WST-RSAMS#36]
URL: https://hcdev.openplus.ca/en/user/203 (Dashboard)
User Agent: User Agent: Chrome, Firefox,
Language: English/French

Issue Description: When a user creates a new recall/alert and clicks on Submit, the focus goes to top of the page. After this, they have to look/scroll for where the new recall/alert is that was just created. 
Also, when the user selects Apply with keyboard by hitting enter, the focus jumps to the next Apply button. There are 4 Apply buttons.

Expected results: The focus should be at the correct location and not jump to top of the page every time a user creates a new alert. 
For Apply button, the focus should also stay where it's at since the user would go over the table that they just applied changes to. 

![image](https://user-images.githubusercontent.com/14280062/128400018-4f3cfce3-10a3-4aed-8241-cc691d641981.png)

![image](https://user-images.githubusercontent.com/14280062/128399930-9586717d-6ecb-4d43-b76b-946b990bf879.png)

**Comment by davidgotlieb** [comment]
Resolved as part of WxT update

**Comment by ndolter** [comment]
Re-tested in both French and English in Chrome v99, and upon creating a new recall, the focus is still going to the top of the page.  For the test involving the Apply button it passes now as the focus is no longer going to the next Apply button.

**Comment by echouin** [comment]
Resolved with latest update.
